modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,254 @@
+<?php
+/*
+ * Simple Payments lets users embed a PayPal button fully integrated with wpcom to sell products on the site.
+ * This is not a proper module yet, because not all the pieces are in place. Until everything is shipped, it can be turned
+ * into module that can be enabled/disabled.
+ * TODO: Once the feature is fully shipped, create a file modules/simple-payments.php with a proper header to turn module on/off.
```

Comment:
After a conversation with @rralian it turned out that there is no plan to make this into a module. The way it's done now will be enough to have this code always available. Keep in mind though that users will still be able to restrict loading these files using filters.